### ISSUE TITLE
Add better error page

### ISSUE THREAD

**Issue by kblief** [SpartaSystems/holdmail#30]
##### Question/Issue Overview

The whitelabel error page shows for all unknown URI. 

##### Expected Behavior

There should be a default /error mapping.
For /rest/unknown-uri there should be a stand error rest response

##### Current Behavior

Access a page that does not exist, whitelabel error page shows

##### Reproducible Sequence
1. Access the application on an invalid URI
** http://holdmail.com/i-dont-exist
** http://holdmail.com/rest/bad-uri 

##### Additional Information

None

**Comment by kinbod** [comment]
Hi,I'm very interested in the project, but it seems a long time no update, I wanna know if it continues？

**Comment by barryoneill** [comment]
Yes, the project is still alive, but as the primary contributor, since I've been busy (and probably will be for another month), I haven't had time to push out the release that I had planned to.  I will; In the meantime, I'm hoping @kblief will be raising a PR for this particular issue soon ;)
